tests/regression/02-base/48-unknown_func_struct.c

```diff
@@ -8,10 +8,8 @@ typedef struct list {
     struct list *next;
 } list_t;
 
-// void mutate_list(list_t n){
-//     list_t *next = n.next;
-//     next->val = 42;
-// }
```

Comment:
Maybe leave this commented out code here, such that one can understand why this invalidation is needed?